ui-components/src/page_components/listing/ListingCard.tsx

```diff
@@ -62,6 +70,21 @@ const ListingCard = (props: ListingCardProps) => {
             </>
           )}
         </div>
+        {profile ? (
+          preferences?.favorites?.includes(props.listingID) ? (
```

Comment:
I'm not super familiar with the backend, but if we define a ManyToMany relation could we calculate on the backend if the listing is favorited or not? This will probably require us to change how we fetch the listings to not be static props, but I think this feature in general makes that true?